contractsSol5/stakingAndDao/KyberDAO.sol

```diff
@@ -117,9 +116,6 @@ contract KyberDAO is IKyberDAO, EpochUtils, ReentrancyGuard, CampPermissionGroup
     // campOptionPoints[campID]: total points and points of each option for a campaign
```

Comment:
reading this again.
for me total votes
is much more clear then total points.

I am not native English.
maybe consult someone else as well.